docs/install_guides/installing-openvino-vcpkg.md

```diff
@@ -48,27 +48,35 @@ Installing OpenVINO Runtime
 
       vcpkg install openvino
 
-   vcpkg  also enables you to install only selected components, by specifying them in the command.
+   vcpkg also enables you to install only selected components, by specifying them in the command.
    See the list of `available features <https://vcpkg.link/ports/openvino>`__, for example: 
 
    .. code-block:: sh
 
-      vcpkg install openvino[cpu,ir]
+      vcpkg install 'openvino[core,cpu,ir]'
+
+   vcpkg also provides a way to install OpenVINO for any specific configuration you want via `triplets <https://learn.microsoft.com/en-us/vcpkg/users/triplets>`, for example to install OpenVINO statically on Windows, use:
```

Comment:
looks like the link has not been rendered well https://docs.openvino.ai/2023.1/openvino_docs_install_guides_installing_openvino_vcpkg.html
Could you please help to fix it?